docs/docs/installation/kustomize.md

```diff
@@ -46,7 +50,48 @@ Else you will get the following error `invalid: metadata.annotations: Too long:
 For example
 
 ```shell
-kubectl replace -k deploy/overlays/cluster_scoped
+kubectl replace -f https://github.com/grafana/grafana-operator/releases/latest/download/kustomize-namespace_scoped.yaml
 ```
 
 For more information how `kubectl replace` works we recommend reading this [blog](https://blog.atomist.com/kubernetes-apply-replace-patch/).
+
+## Kustomize
+
+latest:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# this will automatically pull the latest release when `kustomize build` is executed
+resources:
+  - https://github.com/grafana/grafana-operator/releases/latest/download/kustomize-cluster_scoped.yaml
+```
+
+pinned to release:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# update the version to the release you need
+resources:
+  - https://github.com/grafana/grafana-operator/releases/download/v5.0.10/kustomize-cluster_scoped.yaml
+
+```
+
+### ArgoCD
+
+If you are using ArgoCD you need to add the following patch to fix the errors during apply of the CRD.
+
+```yaml
+patches:
+  - patch: |-
+      apiVersion: apiextensions.k8s.io/v1
+      kind: CustomResourceDefinition
+      metadata:
+        annotations:
+          argocd.argoproj.io/sync-options: Replace=true
```

Comment:
I personally use server side apply instead of replace.

```
metadata:
  annotations:
    argocd.argoproj.io/sync-options: ServerSideApply=true
```

This way the `kubectl.kubernetes.io/last-applied-configuration` isn't created and the CRD don't get too big.
But I guess this is individual, maybe we should provide both suggestions?

Reply:
I added it on my own